apps/writeinpublic/dev.yml

```diff
@@ -0,0 +1,290 @@
+---
+
+- name: Test for docker
+  hosts: writeinpublic-test
+  tasks:
+    - name: Test for docker
+      shell: docker info
+      register: docker_exists
+
+- name: Ensure Docker is installed
+  hosts: writeinpublic-test
+  become: yes
+  vars:
+    docker_package_state: latest
+    docker_users:
+      - dean
+  roles:
+    # ansible-galaxy install geerlingguy.docker
+    - role: geerlingguy.docker
+      when: docker_exists.rc == 1
+
+- name: Ensure Docker Python library is installed
+  hosts: all
+  become: yes
+  vars:
+    pip_executable: pip3
+    pip_package: python3-pip
+    pip_install_packages:
+      - name: docker
+        state: latest
+  roles:
+    # ansible-galaxy install geerlingguy.pip
+    - role: geerlingguy.pip
+      when: docker_exists.rc == 1
+
+#- name: Ensure correct ports are open
+#  hosts: all
+#  vars:
+#    firewall_state: started
+#    firewall_enabled_at_boot: true
+#    firewall_allowed_tcp_ports:
+#      - 22
+#      - 80
+#      - 443
+#    firewall_allowed_udp_ports: []
+#    firewall_disable_ufw: true
+#    firewall_enable_ipv6: false
+#  roles:
+#    # ansible-galaxy install geerlingguy.firewall
+#    - geerlingguy.firewall
+
+- name: Manage Deployment
+  hosts: writeinpublic-test
+  become: no
+  tasks:
+    - name: Write-in-Public Network exists
+      docker_network:
+        name: writeinpublic
+        state: present
+
+#    - name: "Openup Elasticsearch 0.90.13 container exists"
+#      docker_container:
+#        image: openup/elasticsearch-0.90:latest
+
+# Shouldn't be needed once image is pulled from container registry
+    - name: Writeinpublic repo exists
+      git:
+        repo: https://github.com/dnk8n/writeinpublic.git
```

Comment:
Note I am using some local changes I felt necessary in order to move forward. Here is link to a PR to my fork so you can see what I changed - https://github.com/dnk8n/writeinpublic/pull/1/files

Note that there is a local settings at /home/dean/local_settings.py on the test instance. See other comment with line that ensures it is copied to the correct place to be incorporated into the Writeinpublic application.